api/docs/terms_of_service.md

```diff
@@ -0,0 +1,47 @@
+# Openverse API Terms of Service
+
+_Effective as of 4 May 2022_
+
+## 1. Your Agreement to the Terms
+
+Thank you for using the Openverse Catalog application programming interface (“Openverse API”).
+BY USING THE OPENVERSE API, YOU ACKNOWLEDGE THAT YOU HAVE READ, UNDERSTOOD, AND AGREED TO BE BOUND BY THESE API TERMS OF USE (hereafter, “API Terms”).
+By using the Openverse API, you also represent that you have the legal authority to accept the API Terms on behalf of yourself and any party you represent in connection with your use of the Openverse API.
+If you do not agree to the API Terms, you are not authorized to use the Openverse API.
+
+## 2. Licensed Uses and Restrictions
+
+In connection with your use of the Openverse API:
+
+  - You must adhere to all rate limits, registration requirements, and comply with all requirements in the Openverse API documentation;
+  - You must not scrape the content in the Openverse Catalog;
```

Comment:
Why is this?

Reply:
`- You must not scrape the content in the Openverse Catalog`

I'm also not sure what this line means; is it trying to discourage making a bulk copy of _all_ the data supplied by the API?

Reply:
I am honestly not sure. The "catalog" itself isn't even public. I'm happy to delete this line 🤷🏼‍♀️